mautic/mautic
# Improve tooltip of scheduled emails

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

Just small improvement to tooltip scheduled emails. Added date **start sending** at and  **stop sending at**

![image](https://user-images.githubusercontent.com/462477/80312512-ca543c00-87e5-11ea-9c56-1c19693677cf.png)


![image](https://user-images.githubusercontent.com/462477/80312447-92e58f80-87e5-11ea-984a-30ee49fcb75d.png)



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create email with Publish at (date/time) in future and go to email lists and check tooltip
3. Create email with Unpublish at (date/time) in past and go to email list and related check tooltip

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.
